src/map/packet_system.cpp

```diff
@@ -8424,7 +8529,7 @@ void PacketParserInitialize()
     PacketSize[0x03D] = 0x00; PacketParser[0x03D] = &SmallPacket0x03D; // Blacklist Command
     PacketSize[0x041] = 0x00; PacketParser[0x041] = &SmallPacket0x041;
     PacketSize[0x042] = 0x00; PacketParser[0x042] = &SmallPacket0x042;
-    PacketSize[0x04B] = 0x0C; PacketParser[0x04B] = &SmallPacket0x04B;
+    PacketSize[0x04B] = 0x00; PacketParser[0x04B] = &SmallPacket0x04B;
```

Comment:
Does this HAVE to be 0x00 (unchecked) because the size is variable, or can you look up what the size is meant to be?

Reply:
It's because the size is variable depending on the type.  Packet 0x04B is size 0x0C for a regular server message packet, but different size for the ranking packet (I don't remember the actual size ... 0x18 I think?)  I can re-investigate this.